src/main/java/org/kohsuke/github/GitCommit.java

```diff
@@ -0,0 +1,211 @@
+package org.kohsuke.github;
+
+import com.infradna.tool.bridge_method_injector.WithBridgeMethods;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
+import java.util.AbstractList;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * @author Emily Xia-Reinert
+ * @see GHContentUpdateResponse#getCommit() GHContentUpdateResponse#getCommit()
+ */
+
+@SuppressFBWarnings(value = { "NP_UNWRITTEN_FIELD", "UWF_UNWRITTEN_FIELD" }, justification = "JSON API")
+public class GitCommit {
+    private GHRepository owner;
+    private String sha, node_id, url, html_url;
+    private GitUser author;
+    private GitUser committer;
```

Comment:
These are stored as `GHAuthor` in the old code. 

```
        private GHAuthor author;
        private GHAuthor committer;
```

Reply:
I think the reason the coverage went to 0 is that these were changed to `GitUser` and the bridge method that returns `GHAuthor` is never called (requires reflection to be called).

Reply:
You might have to change these back to `GHCommit.GHAuthor` and add clear comments that they should be changed for v2. 

Add `@Deprecated` annotation here (not just the JavaDoc):
https://github.com/hub4j/github-api/blob/eaef661255199d5b0c022e15d70096de0a39b3cd/src/main/java/org/kohsuke/github/GHCommit.java#L75